lib/mayaUsd/render/vp2RenderDelegate/basisCurves.cpp

```diff
@@ -680,6 +676,23 @@ void HdVP2BasisCurves::Sync(
     _UpdateRepr(delegate, reprToken);
 }
 
+void HdVP2BasisCurves::_CommitMVertexBuffer(
+    MHWRender::MVertexBuffer* const buffer,
+    void*                           bufferData) const
+{
+    const MString& rprimId = _rprimId;
+
+    _delegate->GetVP2ResourceRegistry().EnqueueCommit([buffer, bufferData, rprimId]() {
+        MProfilingScope profilingScope(
+            HdVP2RenderDelegate::sProfilerCategory,
+            MProfiler::kColorC_L2,
+            rprimId.asChar(),
+            "CommitBuffer"); // TODO: buffer usage so we know it is positions normals etc
```

Comment:
We have had many code in VP2RenderDelegate that swaps the order of the two string arguments for `MProfilingScope`: `eventName` vs. `description`. But I've recently started to write code with the correct order without changing other existing codes. Maybe we can all start to do the same to avoid inconsistency. Let me know how you think.

Reply:
Looking at MProfilingScope the 3rd argument is meant to be the eventName and the 4th argument is the description. So I think in this case I will switch "CommitBuffer" to be the eventName, and have the rprimId be the description? Is that the more correct way of doing it?

Reply:
I believe so, the new order will make more sense for the text box when we hover the mouse over the event in Profiler.

@kxl-adsk What do you think?